components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/job/scheduler.ts

```diff
@@ -69,55 +69,55 @@ class Job {
 
     constructor(private data) { }
 
-    getName() {
+    public getName(): string {
```

Comment:
Why explicitly set `public`? The default visibility in TypeScript for class members is public